changelogs/unreleased/gh-9112-upgrade-drop-constraint.md

```diff
@@ -0,0 +1,3 @@
+## feature/sql
```

Comment:
Isn't it a breaking change? Should you add the corresponding label to the changelog?